bin/process.sh

```diff
@@ -6,10 +6,15 @@ shift $[$#-2]
 
 input=$1
 output=$2
-# target size in KB
-THUMBNAIL_SIZE=${THUMBNAIL_SIZE:-300}
+
+THUMBNAIL_SIZE=${THUMBNAIL_SIZE:-300} # target size in KB
 TILER_BASE_URL=${TILER_BASE_URL:-http://tiles.openaerialmap.org}
 
+# Although TMPDIR is usually set by the OS, it doesn't seem to be getting passed through
+# Docker and Node's `spawn()`, so no harm done in just defaulting to `/tmp` in such cases.
+# There may be clues here as the underlying issue: https://github.com/npm/npm/issues/4531
+TMPDIR={TMPDIR:-/tmp}
```

Comment:
The only point I'm making is that Docker should be an *option*, not a dependency. For example, as you do in `batch.Dockerfile` here, you use the Docker image as simply an entrypoint to a single command that can be used as if it were a native command. That would be a really nice use of Docker for the refactored transcoder tool - the tool itself is just a BASH (or whatever) script that can run on any POSIX system, then it either depends on a local and native GDAL or a dockerised GDAL. There is nothing about the transcoder itself that means it needs Docker, it is just GDAL, that in certain circumstances (the user doesn't have access to GDAL, or doesn't want to compile it) could possibly benefit from being dockerised.